tests/unit/spec/an_sso.spec.js

```diff
@@ -656,6 +656,9 @@ describe('AsmodeeNet Main Object', function () {
             expect(onFailure).not.toHaveBeenCalled();
             expect(onStoreSet).toHaveBeenCalled();
 
+            var getLimitExp = spyOn(window.AsmodeeNet, 'limit_exp_time').and.callFake(function () {
+                return '1506872982';
+            });
```

Comment:
This is defined and called in two different tests, maybe it could be mutualised? If not, at least the value it returns could be.